dask_kubernetes/core.py

```diff
@@ -525,7 +525,7 @@ async def _start(self):
 
         if not self.scheduler_pod_template:
             self.scheduler_pod_template = base_pod_template
-            self.scheduler_pod_template.spec.containers[0].args = ["dask-scheduler"]
+        self.scheduler_pod_template.spec.containers[0].args = ["dask-scheduler"]
```

Comment:
This breaks functionality for people who want to set custom args for the scheduler.